### PR TITLE
Topic clearwork

### PR DESCRIPTION
**Pull request by CHenrie**

### CODE REVIEW

**Comment by CHenrie** [comment]
@jbest84 Please Review

File: src/_RelatedViewWidgetListMixin.js [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+                scrollerNode,
+                key;
+
+            addView = true;
+            relatedView = action['relatedView'];
+            if (!relatedView.hasOwnProperty('enabled')) {
+                relatedView.enabled = true;
+            }
+
+            if (relatedView && relatedView.enabled) {
```
Comment (jbest84) [inline src/_RelatedViewWidgetListMixin.js]: I would clean up these deep nested ifs by inverting them and do a simple return to bail. You are also checking if relatedView is truthy in this if statement, but above this on line 180 you are calling hasOwnProperty on it without doing a check if it is null or undefined.

File: src/_EditBase.js [hunk 2]
```diff
@@ -675,10 +676,51 @@ define('argos/_EditBase', [
                     content.push(this.sectionBeginTemplate.apply(layout, this));
                 }
 
-                ctor = FieldManager.get(current['type']);
-                field = this.fields[current['name'] || current['property']] = new ctor(lang.mixin({
-                        owner: this
-                    }, current));
+                this.createRowContent(current, content);
+                // ctor = FieldManager.get(current['type']);
```
Comment (jbest84) [inline src/_EditBase.js]: Remove commented out code

File: src/_EditBase.js [hunk 3]
```diff
@@ -675,10 +676,51 @@ define('argos/_EditBase', [
...
+
+                // content.push(template.apply(field, this));
+            }
+
+            content.push(this.sectionEndTemplate.apply(layout, this));
+            sectionNode = domConstruct.toDom(content.join(''));
+            this.onApplySectionNode(sectionNode, current);
+            //domConstruct.place(content.join(''), this.contentNode, 'last');
+            domConstruct.place(sectionNode, this.contentNode, 'last');
+
```
Comment (jbest84) [inline src/_EditBase.js:686]: Extra newline

**Comment by CHenrie** [comment]
@jbest84 Please Re Review after merge

**Comment by jbest84** [comment]
@CHenrie Accept

**Pull request merged**